examples/gno.land/r/sys/vars/gno.mod

```diff
@@ -0,0 +1,10 @@
+module gno.land/r/sys/vars
```

Comment:
please move r/sys/vars to a standalone draft PR

Reply:
Dropped:


30b80f7

I'll open a PR soon 👍